src/main/java/com/mfahproj/webapp/handlers/ReportHandler.java

```diff
@@ -174,12 +177,59 @@ else if (type.equals("EAR")) {
 
         }
 
+
+        // !!!!!!!!!!!!!!!!!!!!!!!!!!!!
+        else if (type.equals("MDR")) {
+
+            StringBuilder query = new StringBuilder();
+            query.append("SELECT SUM(CASE WHEN age <= 12 THEN 1 ELSE 0 END) AS Children,");
+            query.append(" SUM(CASE WHEN age BETWEEN 13 AND 19 THEN 1 ELSE 0 END) AS Teens,");
+            query.append(" SUM(CASE WHEN age BETWEEN 20 AND 54 THEN 1 ELSE 0 END) AS Adults,");
+            query.append(" SUM(CASE WHEN age >= 55 THEN 1 ELSE 0 END) AS Seniors");
+            query.append(" FROM (");
+            query.append(" SELECT");
+            query.append(" DATEDIFF(CURDATE(), birth_date) / 365 AS age\"");
```

Comment:
This line should be: `query.append(" DATEDIFF(CURDATE(), birth_date) / 365 AS age");`
The extra `\"` was causing an issue with the query.

Reply:
Also `birth_date` needs to be `BirthDate`